gsy_framework/sim_results/electric_blue/volume_timeseries.py

```diff
@@ -127,8 +127,10 @@ def _get_asset_volume_time_series(self, time_slot):
             self._asset_volume_time_series_buffer[time_slot] = time_series
         return time_series
 
-    def _fetch_asset_volume_time_series_from_db(self, time_slot: DateTime):
-        # TODO: should be implemented by Kamil.
+    def _fetch_asset_volume_time_series_from_db(
+            self, time_slot: DateTime) -> Union[Dict[DateTime, Dict], None]:
```

Comment:
Nit: I think the following is correct:
```python
    -> Optional[Dict[DateTime, Dict]]:
```

Reply:
They are really the same but sure I'll do it in order to have the same convention in the whole repo. Thanks